src/main/resources/db/migration/common/V1_416__ArmMissingResponseReplay_automated_task.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable, batch_size,
+                                  created_ts, created_by,last_modified_ts,last_modified_by, task_enabled)
+VALUES (33,'ArmMissingResponseReplay','Replay objects for Missing ARM response files','0 0 0 31 2 ?',true,10000,
+        current_timestamp, 0 , current_timestamp,0, false);
```

Comment:
Minor thing but formatting is a bit on the wonk with whitespace around the commas.

Reply:
Updated :)